third_party/rust/chromium_crates_io/vendor/bitflags-2.6.0/Cargo.lock

```diff
@@ -0,0 +1,383 @@
+# This file is automatically @generated by Cargo.
```

Comment:
this is odd, I wonder why this was added if bitflags is already vendored and no version changes. cc @cdesouza-chromium

Reply:
This file can be deleted. I've rebased this branch and ran `gnrt` and it seems to be all good. This PR needs to be rebased anyway, and gnrt has to be rerun, because otherwise we will have a dirty gnrt run tree in master. I would advise to delete this folder, and run gnrt with this PR on master.